backend/src/appointment/routes/subscriber.py

```diff
@@ -24,12 +24,20 @@ def get_all_subscriber(db: Session = Depends(get_db), _: Subscriber = Depends(ge
 @router.put("/disable/{email}")
 def disable_subscriber(email: str, db: Session = Depends(get_db), _: Subscriber = Depends(get_admin_subscriber)):
     """endpoint to disable a subscriber by email, needs admin permissions"""
-    # TODO: Add status to subscriber, and disable it instead.
-    raise NotImplementedError
+    subscriber = repo.subscriber.get_by_email(db, email)
+    if not subscriber:
```

Comment:
Can we prevent the disabling of already disabled subscribers and the subscriber themself? 😄

Reply:
Of course! I added validation for this.